pkg/registry/core/pod/strategy_test.go

```diff
@@ -1189,8 +1180,12 @@ func TestPodStrategyValidate(t *testing.T) {
 
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			if errs := Strategy.Validate(genericapirequest.NewContext(), tc.pod); len(errs) != 0 {
-				t.Errorf("unexpected error:%v", errs)
+			errs := Strategy.Validate(genericapirequest.NewContext(), tc.pod)
+			if tc.wantErr && len(errs) == 0 {
+				t.Errorf("expected errors but gotten none")
```

Comment:
tiny nit:
```suggestion
				t.Errorf("expected errors but got none")
```
(gotten is an odd tense here)

Reply:
updated and squashed.